.code-samples.meilisearch.yaml

```diff
@@ -551,6 +540,131 @@ getting_started_search_md: |-
   ```
 
   [About this SDK](https://github.com/meilisearch/meilisearch-rust/)
+getting_started_update_rankingRules: |-
+  let ranking_rules = [
+    "exactness",
+    "words",
+    "typo",
+    "proximity",
+    "attribute",
+    "sort",
+    "release_date:asc",
+    "rank:desc"
+  ];
+
+  client.index("movies").set_ranking_rules(&ranking_rules).await.unwrap();
+getting_started_update_searchableAttributes: |-
+  let searchable_attributes = [
+    "title"
+  ];
+
+  client.index("movies").set_searchable_attributes(&searchable_attributes).await.unwrap();
+getting_started_update_stop_words: |-
+  let stop_words = ["the"];
+  client.index("movies").set_stop_words(&stop_words).await.unwrap();
+getting_started_check_task_status: |-
+  client.index("movies").get_task(0).await.unwrap();
+getting_started_synonyms: |-
+  let mut synonyms = std::collections::HashMap::new();
+  synonyms.insert(String::from("winnie"), vec![String::from("piglet")]);
+  synonyms.insert(String::from("piglet"), vec![String::from("winnie")]);
+
+  client.index("movies").set_synonyms(&synonyms).await.unwrap();
+getting_started_update_displayedAttributes: |-
+  let displayed_attributes = [
+    "title",
+    "overview",
+    "poster",
+  ];
+
+  client.index("movies").set_displayed_attributes(&displayed_attributes).await.unwrap();
+getting_started_communicating_with_a_protected_instance: |-
+  let client = Client::new("http://localhost:7700", "apiKey");
+  client.index("movies").search()
+getting_started_add_meteorites: |-
+  use serde::Deserialize;
+  use std::fs::File;
+
+  #[derive(Serialize, Deserialize, Debug)]
+  struct Geo {
+    lat: f64,
+    lon: f64
+  }
+
+  struct Meteorite {
+    name: String,
+    id: String,
+    nametype: String,
+    recclass: String,
+    mass: i64,
+    fall: String,
+    _geo: Geo
+  }
+
+  impl Document for Meteorite {
+    type UIDType = String;
+    fn get_uid(&self) -> &Self::UIDType { &self.id }
+  }
+
+  let mut file = File::open("meteorites.json")?;
+  let meteorites: Vec<Meteorite> = serde_json::from_reader(file)?;
+
+  client.index("meteorites").add_documents(&meteorites, None).await?;
+getting_started_configure_settings: |-
+  let mut synonyms = std::collections::HashMap::new();
+  synonyms.insert(String::from("wolverine"), vec!["xmen", "logan"]);
+  synonyms.insert(String::from("logan"), vec!["wolverine"]);
+
```

Comment:
```suggestion
```

Reply:
:+1: 
And from what I see the other sdk are not setting the `searchable_attributes` nor the `displayed_attributes`.